src/providers/test-report-validator/__tests__/test-report-validator.spec.ts

```diff
@@ -38,14 +51,18 @@ describe('TestReportValidator', () => {
       const result = testReportValidatorProvider.isTestReportValid({}, TestCategory.BE);
       expect(result).toEqual(false);
     });
-    it('should return true if the test report is valid for a Cat C test', () => {
-      const result = testReportValidatorProvider.isTestReportValid(validTestCatBE, TestCategory.C);
-      expect(result).toEqual(true);
-    });
-    it('should return false if the test report is not valid for a Cat C test', () => {
-      const result = testReportValidatorProvider.isTestReportValid({}, TestCategory.C);
-      expect(result).toEqual(false);
+
+    cCats.forEach((cat) => {
+      it(`should return true if the test report is valid for a Cat ${cat.category} test`, () => {
+        const result = testReportValidatorProvider.isTestReportValid(cat.validTest, cat.category);
+        expect(result).toEqual(true);
+      });
+      it(`should return false if the test report is not valid for a Cat ${cat} test`, () => {
+        const result = testReportValidatorProvider.isTestReportValid({}, cat.category);
+        expect(result).toEqual(false);
```

Comment:
Really like this - was looking for how to run multiple things through the same test like other testing frameworks.